lib/room.rb

```diff
@@ -0,0 +1,23 @@
+module Room
+  NUM_OF_ROOMS = 20
+  COST_PER_NIGHT = 200
+
+  def self.make_rooms
+    rooms_res_hash = {}
+    (1..NUM_OF_ROOMS).each { |i| rooms_res_hash[i] = [] }
```

Comment:
Interesting a hash of rooms with the number being the key, and the value being an array of reservations.  Nice!